themes/book/app/routes/[custom-style.css].tsx

```diff
@@ -0,0 +1,16 @@
+import type { LoaderFunction } from '@remix-run/node';
+import { getCustomStyleSheet } from '~/utils/loaders.server';
+
+export const loader: LoaderFunction = async (): Promise<Response> => {
+  const css = await getCustomStyleSheet();
+  if (!css) {
+    return new Response(`/* No custom stylesheet found */`, {
```

Comment:
@rowanc1 what do you think about this? 

The issue is that AFAICT we can't define the `links` function such that it only adds custom CSS if it's defined. So, rather than 404ing for any site that doesn't deploy custom CSS, the "work around" is to define an empty fall-back. Is this OK?

Reply:
Yeah, I think this is good. An extra network request, but not a big deal.